packages/page-bounties/src/types.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2020 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+import type { AccountId, BlockNumber } from '@polkadot/types/interfaces/runtime';
```

Comment:
```suggestion
// SPDX-License-Identifier: Apache-2.0

import type { AccountId, BlockNumber } from '@polkadot/types/interfaces/runtime';
```

I guess the header rule needs to check for a newline.

Reply:
Yes, well, it doesn't support empty-line-following